ibc/store/provable_store.go

```diff
@@ -37,20 +41,23 @@ func (c *cachedEntry) prepare() (key, value []byte) {
 // the creation of IBC messages that are broadcasted through the network and
 // included in the mempool/next block to change the state of the IBC tree
 type provableStore struct {
-	m      sync.Mutex
-	bus    modules.Bus                // used to interact with persistence (passed from IBCHost)
-	name   string                     // store name in storeManager
-	prefix coreTypes.CommitmentPrefix // []byte(name)
-	cache  map[string]*cachedEntry    // in-memory cache of local changes to be written to disk
+	m          sync.Mutex
+	bus        modules.Bus                // used to interact with persistence (passed from IBCHost)
```

Comment:
Order for clarity.

1. Bus at top
2. Mutex next to the object (i.e. cache?) it's responsible for protecting
3. Add comment to the utex